Remove grid-width mixin (#1342)
alphagov/govuk-frontend: CHANGELOG.md

  ([PR #1330](https://github.com/alphagov/govuk-frontend/pull/1330))

- Remove grid-width mixin

  https://github.com/alphagov/govuk-frontend/pull/1090 deprecated this mixin
  and left it in as an alias to govuk-grid-width.

>   **ACTION TO BE TAKEN**: Replace all instances of `grid-width` mixin with `govuk-grid-width`

Reviewer: Again, not sure about this.

```suggestion
  To migrate you'll need to replace any references to the `grid-width` mixin with `govuk-grid-width`
```

> Closes https://github.com/alphagov/govuk-frontend/issues/1091

Reviewer: We've never put this in before. Do we think this is useful for end users?

Author: Ah, thats not meant to be in there. 👍 good spot
